### ISSUE TITLE
Wayland support?

### ISSUE THREAD

**Issue by acro5piano** [genya0407/rumap#18]
Just curious, are you going to support Wayland? I'm trying to implement it, but no luck for now.

https://github.com/acro5piano/sway-remap

**Comment by genya0407** [comment]
No, because I do not use Linux desktop now, and I don't know much about Wayland...
